src/styles/utils/variables.scss

```diff
@@ -0,0 +1,4 @@
+$width-boxs: 100px;
+$width-gap: 1px;
+$calendar-days: 31;
+$my-var: $width-boxs * 7 + 6 * $width-gap;
```

Comment:
The variable name `$my-var` is not very descriptive. Try to use a more meaningful name, which reflects the purpose of the variable. For instance, if this variable is used to calculate the total width of the calendar, you could name it `$calendar-total-width`.